src/Hazelcast.Net.Tests/Core/SocketExtensionsTests.cs

```diff
@@ -78,7 +78,7 @@ public async Task ConnectAsyncConnectionRefused1()
         [Test]
         public async Task ConnectAsyncConnectionRefused2()
         {
-            var endpoint = IPEndPointEx.Parse("127.0.0.1:11000");
+            var endpoint = IPEndPointEx.Parse("127.0.0.1:11002");
```

Comment:
The socket listener is already going to randomize / offset the port - how is this fixing?

Reply:
I didn't see that they are randomized in the socket listener. Now, I reverted these port values. In this case, the test is not touched. After all test fixes merged, we may need going over it one more time if there is need.

Reply:
ok